WcaOnRails/app/webpacker/components/StaticPages/Logo.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import I18n from '../../lib/i18n';
+import I18nHTMLTranslate from '../I18nHTMLTranslate';
+import '../../stylesheets/static_pages/logo.scss';
+
+/**
+ * @returns {JSX.Element}
+ * @constructor
+ */
+function Logo({ title }) {
+  return (
+    <div className="wca-logo-information">
+      <h1>{title}</h1>
+      {Object.keys(I18n.t('logo.paragraphs')).map((index) => (
```

Comment:
nit: I would rename `index` to `key`. `index` implies a numerical index value which does not appear to be happening here.